### PR TITLE
Updated README and test instructions

### PR DESCRIPTION
**Pull request by pcostell**
- Converted to better markdown style.
- Removed a lot of obsolete info from README.
- Updated tox to run tests in a single command, rather than one for tests and one for actual module.

### CODE REVIEW

**Comment by pcostell** [comment]
@eddavisson -- PTAL.

File: README.md [hunk 1]
```diff
@@ -1,81 +1,39 @@
...
-Eventually, when the project is sufficiently mature, the code will
-(hopefully) become a standard component of the App Engine Python SDK
-and the Python runtime. Until then, however, the way to use this code
-is to check it out from the repository and copy it into your
-application.
+---
+**Note:** As of App Engine SDK 1.6.4, ndb on App Engine has reached status
+General Availability.  
+    
+Using NDB from outside of App Engine through Cloud Datastore v1beta3 is still in the pre-launch phase.
```
Comment (eddavisson) [inline README.md]: s/NDB/ndb/?

File: README.md [hunk 2]
```diff
@@ -1,81 +1,39 @@
...
-to worry about backwards compatibility between versions of this
-experimental code. You are not required to check out the trunk,
-however I do not plan to fix bugs in older branches.
+---
 
-See the file LICENSE for the open source licensing conditions (which
-are the same as for the App Engine SDK).
+ndb is a client library for use with [Google Cloud Datastore][0].
+It was designed specifically to be used from within the 
+[Google App Enigine][1] python runtime.
```
Comment (eddavisson) [inline README.md]: Engine

File: README.md [hunk 3]
```diff
@@ -1,81 +1,39 @@
...
 
+    from google.appengine.ext import ndb
 
-Overview
---------
+It is also possible to include ndb directly from this github project.
+This will allow application developers to manage their own dependencies. Note
+however that ndb depends on the non-public Datastore App Engine RPC API. This
+means that there is no explicit support for older versions of ndb in the 
+App Engine python runtime.
```
Comment (eddavisson) [inline README.md]: Python

File: README.md [hunk 4]
```diff
@@ -1,81 +1,39 @@
...
 
---Guido van Rossum <guido@google.com>
+ndb is included in the python runtime and is available through a
+standard python import.
 
+    from google.appengine.ext import ndb
 
-Overview
---------
+It is also possible to include ndb directly from this github project.
```
Comment (eddavisson) [inline README.md]: GitHub

File: README.md (hunk 2 above)
Comment (eddavisson) [inline README.md]: Python

File: tox.ini [hunk 5]
```diff
@@ -35,12 +35,4 @@ commands =
...
-    logilab-common<=0.63.0  # Hack for pylint upstream
-
-[testenv:lint-test]
-basepython =
-    python2.7
-commands =
-    {[testenv:lint]commands} --test
-deps = 
-    {[testenv:lint]deps}
+    logilab-common<=0.63.0  # Hack for pylint upstream
```
Comment (eddavisson) [inline tox.ini]: Missing newline

**Comment by eddavisson** [comment]
LGTM

**Comment by eddavisson** [comment]
LGTM

**Pull request merged**